src/main/resources/assets/js/app/settings/browse/SettingsBrowsePanel.ts

```diff
@@ -32,6 +34,27 @@ export class SettingsBrowsePanel
         ProjectChangedEvent.on(projectSetHandler);
     }
 
+    protected initListeners(): void {
+        super.initListeners();
+
+        this.treeGrid.onDataChanged((event: DataChangedEvent<SettingsViewItem>) => {
+            const previewItemId: string = !!this.getBrowseItemPanel().getStatisticsItem()
+                                          ? this.getBrowseItemPanel().getStatisticsItem().getModel().getId()
+                                          : null;
```

Comment:
You could do 

`const previewItemId: string = this.getBrowseItemPanel().getStatisticsItem()?.getModel().getId();`